docs/alerts_api.rst

```diff
@@ -0,0 +1,275 @@
+         -d '
+    {
+        'severity': 5,
+        'summary': 'mfbt now!',
+        'description': 'it is 4:00pm on a friday and it is mfbt.',
+        'emitter_name': 'mfbt-watcher',
+        'emitter_version': 1
+    }'
+
+
+Using Python requests:
+
+.. code-block:: python
+
+    import json
+    import requests
+
+    payload = {
+        'severity': 5,
+        'summary': 'mfbt now!',
+        'description: 'it is 4:00pm on a friday and it is mfbt.',
+        'emitter_name': 'mfbt-watcher',
+        'emitter_version': 1
+    }
+    headers = {
+        'content-type': 'application/json',
+        'accept': 'application/json; indent=4',
+        'authorization': 'token cd64de0e6c4c491f90fe1d362104c1e5',
+    }
+    resp = requests.post(
+        'https://input.mozilla.org/api/v1/alerts/mfbt/',
+        data=json.dumps(payload),
+        headers=headers
+    )
+
+    print resp.status_code
+    # 201
+    print resp.json()
+    # something like {u'detail': {u'id': 4}}
+
+
+Required fields
+---------------
+
+These fields are required and have no defaults. If you do not provide
+them, then you'll get back an HTTP 400 with a message stating you
+missed a required field.
+
++-------------------+-------+--------------------------------------------------------+
+|field              |type   |notes                                                   |
++===================+=======+========================================================+
+|severity           |integer|This is the severity of the alert. 0 = don't care. 10 = |
+|                   |       |call the president.                                     |
++-------------------+-------+--------------------------------------------------------+
+|summary            |string |Brief summary of what the alert is about.               |
++-------------------+-------+--------------------------------------------------------+
+|description        |string |Involved description fo all the details that help you   |
+|                   |       |understand what this alert is about.                    |
+|                   |       |                                                        |
+|                   |       |You can make this a JSON encoded field if you have lots |
+|                   |       |of key/value pairs you want to include.                 |
++-------------------+-------+--------------------------------------------------------+
+|emitter_name       |string |Max length: 100.                                        |
+|                   |       |                                                        |
+|                   |       |The name of the emitter that created this alert. It     |
+|                   |       |could be a script name. It could be a sekret code-name  |
+|                   |       |for the emitter. It could be skynet. So long as we can  |
+|                   |       |distinguish it from other emitters, it's all good.      |
```

Comment:
I was super tired writing this and dealing with emacs table-mode. I'm totally game for anything you have in mind.